packages/server/graphql/mutations/jiraCreateIssue.ts

```diff
@@ -52,13 +55,33 @@ export default {
     // VALIDATION
     const viewerAuthRes = await dataLoader.get('atlassianAuthByUserId').load(viewerId)
     const viewerAuth = viewerAuthRes.find((auth) => auth.teamId === teamId)
-
     if (!viewerAuth || !viewerAuth.isActive) {
       return standardError(new Error('The viewer does not have access to Jira'), {
         userId: viewerId
       })
     }
 
+    if (meetingId) {
+      const meeting = (await r
```

Comment:
`teamId` is stored on the meeting member, so you could remove this DB hit & just go off the meetingMember